app/src/main/java/com/example/lawn_care/editYourProperty.java

```diff
@@ -89,17 +104,410 @@ protected void onCreate(Bundle savedInstanceState) {
+            case "PA":
+                spinnerState.setSelection(39,true);
+                break;
+            case "RI":
+                spinnerState.setSelection(40,true);
+                break;
+            case "SC":
+                spinnerState.setSelection(41,true);
+                break;
+            case "SD":
+                spinnerState.setSelection(42,true);
+                break;
+            case "TN":
+                spinnerState.setSelection(43,true);
+                break;
+            case "TX":
+                spinnerState.setSelection(44,true);
+                break;
+            case "UT":
+                spinnerState.setSelection(45,true);
+                break;
+            case "VT":
+                spinnerState.setSelection(46,true);
+                break;
+            case "VA":
+                spinnerState.setSelection(47,true);
+                break;
+            case "WA":
+                spinnerState.setSelection(48,true);
+                break;
+            case "WV":
+                spinnerState.setSelection(49,true);
+                break;
+            case "WI":
+                spinnerState.setSelection(50,true);
+                break;
+            case "WY":
+                spinnerState.setSelection(51,true);
+                break;
+            default:
+                spinnerState.setSelection(0,true);
+        }
+
+        ET_address.setText(addressEdit);
+        ET_city.setText(cityEdit);
+        ET_zipcode.setText(zipEdit);
+        ET_PropertySize.setText(lawnSizeEdit);
+        ET_JobType.setText("");
+        ET_Tools.setText("");
+
+
+        if (workNeededEdit.contains("Clippings") == true)
+            CB_clippings.setChecked(true);
+        else
+            CB_clippings.setChecked(false);
+        if (workNeededEdit.contains("Trimming") == true)
+            CB_trimming.setChecked(true);
+        else
+            CB_trimming.setChecked(false);
+        if (workNeededEdit.contains("Mowing") == true)
+            CB_mowing.setChecked(true);
+        else
+            CB_mowing.setChecked(false);
+        if (workNeededEdit.contains("Fertilization") == true)
+            CB_fertilization.setChecked(true);
+        else
+            CB_fertilization.setChecked(false);
+        if (workNeededEdit.contains("Weed_Control") == true)
+            CB_weed_Control.setChecked(true);
+        else
+            CB_weed_Control.setChecked(false);
+        if (workNeededEdit.contains("Pest_Control") == true)
+            CB_pest_Control.setChecked(true);
+        else
+            CB_pest_Control.setChecked(false);
+        if (workNeededEdit.contains("Irrigation") == true)
+            CB_irrigation.setChecked(true);
+        else
+            CB_irrigation.setChecked(false);
+        if (workNeededEdit.contains("Aeration") == true)
+            CB_aeration.setChecked(true);
+        else
+            CB_aeration.setChecked(false);
+
     }
 
     public void ResetAll(View view) {
```

Comment:
what does this function do? shouldn't it just reset

Reply:
Yes its's just reset all the fields with Spinner